cmd/thanos/sidecar.go

```diff
@@ -281,6 +286,20 @@ func runSidecar(
 	return nil
 }
 
+func isPrometheusDirAccesible(dir string) error {
+	const errMsg = "WAL file is not accessible. Does block shipper dir is a TSDB directory? If yes it is shared with TSDB?"
+	f, err := os.Stat(filepath.Join(dir, "WAL"))
+	if err != nil {
+		return errors.Wrap(err, errMsg)
+	}
+
+	if f.IsDir() {
```

Comment:
`WAL` is a directory ... dont think we should error here.

Reply:
`if !f.IsDir() {`

Reply:
Wut? I guess I might have fixed it in another PR.. will fix it here, sorry